src/components/pages/Stripe/StripeCheckoutButton.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import StripeCheckout from 'react-stripe-checkout';
+//import axiosWithAuth from '../../../utils/axiosWithAuth';
+import axios from 'axios';
+
+const StripeCheckoutButton = ({ price }) => {
+  const priceForStripe = price * 100;
+  const publishablekKey =
+    'pk_test_51KW2lbBKjoE9XTUg0egVFw0BOv1qLHRAalJAI0tzcCrPQZiuT33aJjmlKJkDp0FrfvbcY6JR2Pl7I2tSukYUyQTk00OSmOo3M2';
+
+  const onToken = token => {
+    // axiosWithAuth().post('/payments/payments', {
```

Comment:
We can probably take this out if we are not using axios with auth for this request